content/en/community/hackathons/sessions/bincompat/content/Work-Items_02.-Build-app-elfloader.md

```diff
@@ -1,40 +1,76 @@
-Using `do.sh run` we ran the prebuilt images from [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
+Using `./run*.sh` we ran the prebuilt images from [the `run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
 Let's now also build `app-elfloader`.
 
-Run the following commands:
+Run the following commands, similar to what we did in previous sessions when building usual applications:
 
 ```console
-$ ./do.sh clean
-
-$ ./do.sh setup
-
-$ ./do.sh build
-
-$ ./do.sh run_built ...
+$ git clone https://github.com/unikraft/app-elfloader/
```

Comment:
Remove dollar sign.

I would add these to the `app-elfloader` repository as part of its README. And simply reference that README, without duplicating the information. We would amend [this PR](https://github.com/unikraft/app-elfloader/pull/22/files).